docs/diagrams/shion/SessionClassDiagram.puml

```diff
@@ -0,0 +1,29 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR_T1
+
+show members
+
+  namespace seedu.address {
+    namespace model {
```

Comment:
I am not entirely sure if the inclusion of the `seedu.address` and `model` boxes and much value here. Maybe the rest can look into this too!

Reply:
Yeah, I think as per the tutorial yst, having a class diagram of a single class might not add value in this case.

Reply:
I'll just leave it in there n not use it

Reply:
can use it one day in the components perhaps